docs/user-guide/join-a-jitsi-meeting.md

```diff
@@ -4,6 +4,11 @@ title: Join a Jitsi Meeting
 sidebar_label: Join a Jitsi Meeting
 ---
 
-Welcome to the user guide!
 
-Check back soon!
+1. When visiting a Jitsi Meeting link, your browser may first ask you to grant microphone and/or camera access.
+   Please refert to the browser's documentation for details (e.g.
+   [Firefox](https://support.mozilla.org/kb/how-manage-your-camera-and-microphone-permissions#w_using-prompts-to-allow-or-block-camera-and-microphone-permissions-for-a-site), 
+   [Chrome](https://support.google.com/chrome/answer/2693767)).
+1. Enter a name. This will be visible to other participants in the Jitsi Meeting room.
```

Comment:
1. Regarding the numbering: Could you use `2., 3.` etc. instead? This makes understanding the "source code" for others easier.

2. Small changes I would recommend:
`Enter a name, it will be visible to other participants in the Jitsi Meeting room.`